blaseball_mike/models.py

```diff
@@ -1215,6 +1215,24 @@ def player(self):
         return self._player
 
 
+class Tribute(Base):
+
+    @classmethod
+    def load(cls):
+        tributes = database.get_tributes()
+        tributes_dict = OrderedDict()
+        for tribute in tributes:
+            tributes_dict[tribute['playerId']] = cls(tribute)
+        return tributes_dict
+
+    @property
```

Comment:
you need to declare the setter

Reply:
`Idol` also doesnt have a setter for player, should I add one there as well?

Reply:
OK, added for Idol and Tribute